dev/com.ibm.ws.jpa_spec10_query_fat/publish/servers/JPA10QueryServer/database.xml

```diff
@@ -10,18 +10,30 @@
  -->
  <server>
     <dataSource id="jdbc/JPA_DS" jndiName="jdbc/JPA_DS" fat.modify="true">
-            <jdbcDriver libraryRef="JDBCLib"/>
-            <properties.derby.embedded databaseName="memory:ds1" createDatabase="create" user="dbuser1" password="{xor}Oz0vKDtu" />
+            <jdbcDriver libraryRef="AnonymousJDBCLib"/>
+            <properties.derby.embedded databaseName="memory:ds1" createDatabase="create" />
     </dataSource>
 
     <dataSource id="jdbc/JPA_NJTADS" jndiName="jdbc/JPA_NJTADS" fat.modify="true" transactional="false">
-            <jdbcDriver libraryRef="JDBCLib"/>
-            <properties.derby.embedded databaseName="memory:ds1" createDatabase="create" user="dbuser1" password="{xor}Oz0vKDtu" />
+            <jdbcDriver libraryRef="AnonymousJDBCLib"/>
+            <properties.derby.embedded databaseName="memory:ds1" createDatabase="create" />
```

Comment:
Because the database tooling will replace it.  I followed the example from `com.ibm.ws.concurrent.persistent_fat_demo_timers`.  Having it there is the correct thing to do.